agents/plugins/smart

```diff
@@ -11,6 +11,12 @@
 # shellcheck disable=SC2034
 CMK_VERSION="2.1.0i1"
 
+# Function to replace "if type [somecmd]" idiom
```

Comment:
Its better to avoid checking if the `udevadm` command exists for every loop iteration. Instead, we can do something like
`INPATH_UDEVADM=$(command -v udevadm)`

Reply:
Is there already a standard for cases like that?
I went for another approach.

Reply:
I havent found a case where `inpath` is used inside a loop, which is what I want to avoid.